client/app/components/Topbar/topbar.css

```diff
@@ -0,0 +1,14 @@
+.topbar {
+  display: flex;
+  flex-direction: row;
+  height: 80px;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--colorBackground);
+  border-bottom: 1px solid rgba(0, 0, 50, 0.1);
+}
+
+.topbarLogo {
+  height: 40px;
```

Comment:
Would it make sense to have this `40px` and the `80px` heights in the variables?